We need to use readlink on argv[0] so that the script gets the correct
prefix

Signed-off-by: Jonas Karlsson <jonas@gobolinux.org>
--- albumart-1.6.2/bin/albumart-qt	2005-02-21 20:58:31.000000000 +0100
+++ albumart-1.6.2/bin/albumart-qt.new	2008-03-24 19:01:24.000000000 +0100
@@ -17,7 +17,7 @@
 # @returns the directory where the script resides
 #
 def getBaseDir():
-  path = os.path.dirname(sys.argv[0])
+  path = os.path.dirname(os.readlink(sys.argv[0]))
   if os.path.split(path)[-1] == 'bin':
     path = os.path.dirname(path)
   return path
